content/how-to/ignore-rules/global/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Configuring global ignore rules"
-description: "How to Set Ignore Rules Globally for Tenant"
-lead: "How to Ignore Rules Globally for Tenant"
+title: "How to Set Ignore Rules Globally for Tenant"
```

Comment:
Did you mean to make the title sentence case?

Reply:
I did, I'm returning the title back as well.